src/DynamoCoreWpf/ViewModels/Search/SearchCategory.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Dynamo.Search.SearchElements;
+using Dynamo.UI;
+using Dynamo.Wpf.ViewModels;
+
+namespace Dynamo.Search
+{
+    public class SearchCategory
```

Comment:
If the `TODO` comments in this file are not required, please remove them. If they are still needed, please create an internal task to track the progress, then add the link to that task alongside the `TODO` comment in the source code.